lib/core/VersionManager.ts

```diff
@@ -125,12 +115,7 @@ export default class VersionManager implements IVersionManager, IVersionMetadata
    */
   public getBatchWriter (blockchainTime: number): IBatchWriter {
     const version = this.getVersionString(blockchainTime);
-    const batchWriter = this.batchWriters.get(version);
-
-    if (batchWriter === undefined) {
-      throw new SidetreeError(CoreErrorCode.VersionManagerBatchWriterNotFound, `Batch writer for blockchain time ${blockchainTime} not found.`);
-    }
-
+    const batchWriter = this.batchWriters.get(version)!;
```

Comment:
Do we not care if we ask for a bad version?

Reply:
nvm, the error is thrown in getVersionString now, I like it!

Reply:
Sanity: `getVersionString()` in this class also throws the same error?